CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# Contributing to OSIAM
+Thank you for considering to contribute to OSIAM.
+
+This document is here to offer you a few helpful notes to get you started and
+make your experience as smooth as possible. They should be seen as guidelines.
+They are _not_ strict rules and we won't blame anyone who tries to contribute.
+By following the points in this document you will make our lifes a bit easier
+but we will still be happy about your contribution if you don't.
+
+## Reporting issues on Github
+The easiest way to contribute to OSIAM is by filing issues on Github. It does
+not have to be a bug, it can also be a feature request or a suggestion to
+improve our documentation. We will do our best to answer your issue as soon 
+as possible. Before you create an issue on Github please consider doing the
+following:
+
+* Search existing issues before raising a new one. 
+* If you are reporting a bug please include as much details as possible.
+  This might include:
+    * Which OS you're using
+    * Which version you've been running
+    * Logs/Stacktrace of the error that occurred
+    * Steps to reproduce
```

Comment:
When reporting an issue it might be good If this will be done with the following structure:
- Steps to reproduce
- Expectations
- Actual

@tkrille already did a nice job when he reported [this issue](https://github.com/osiam/addon-administration/issues/81)